Modules/SalesAutomation/Resources/views/sales-area/index.blade.php

```diff
@@ -52,8 +53,43 @@
 						</form>
 					</td>
 				</tr>
-			@endforeach		
-	    </thead>
+				@empty
+				<tr>
+                    			<td colspan="2">
+						<div class="d-flex justify-content-center">
+							<img src="{{ URL('images/no-result.png') }}" class='img-center' alt="" width="50%">
```

Comment:
Remove the unwanted attributes.
```suggestion
							<img src="{{ URL('images/no-result.png') }}" class='img-center' width="50%">
```